coffee-shop/index.html

```diff
@@ -0,0 +1,87 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
```

Comment:
Please don't modify this folder and please make sure all your changes would be inside `kundan` folder.